Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Buttons/ButtonConfigHelper.cs

```diff
@@ -0,0 +1,418 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Per [pull request guidelines](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/PullRequests.html#creating-a-pull-request), please update documentation to describe how to use the button configurator. I think updating the Button documentation page would work (it would be good to work with @cre8ivepark on this).